src/lib-python/CMakeLists.txt

```diff
@@ -1,22 +1,49 @@
-list( APPEND programs
+list( APPEND libs
   ioda_conv_ncio.py
   collect_sources.py
+)
+
+list( APPEND programs
   run-pyflakes.py
   run-mccabe.py
 )
 
-SET_TARGETS_DEPS( "${programs}"
+SET_TARGETS_DEPS( "${libs}"
                    ${CMAKE_CURRENT_SOURCE_DIR}
-                   ${CMAKE_BINARY_DIR}
+                   ${CMAKE_BINARY_DIR}/lib
```

Comment:
What are everyone's thoughts on having subdirectories in lib/? Would it be better to have marine/, gsi-diag/, etc. in here or everything in one lib/ top level directory?

Reply:
My opinion is to have subdirectories with relevant names as you mentioned ... marine/ etc

Reply:
Would it make sense to have a single subdirectory (py-iodaconv for example) in which we place all of the library type scripts? It doesn't seem that we have that many scripts, and this would keep them contained away from all of the .a and other files.

Reply:
@aerorahul, I agree that ultimately the library scripts should go in the standard python directory (lib/python2.7/site-packages/pyiodaconv). I also think it's a good idea to approach this incrementally, and that your scheme using lib/pyiodaconv is a good first step.

Thanks for doing this work and let's move forward with this plan.